include/heuristic/HeuristicMapper.hpp

```diff
@@ -106,7 +100,10 @@ class HeuristicMapper : public Mapper {
      * @brief applies an in-place swap of 2 qubits in `qubits` and `locations`
      * of the node
      */
-    void applySWAP(const Edge& swap, Architecture& arch);
+    void applySWAP(const Edge& swap, Architecture& arch,
+                   const SingleQubitMultiplicity& singleQubitGateMultiplicity,
+                   const TwoQubitMultiplicity&    twoQubitGateMultiplicity,
+                   bool                           considerFidelity);
```

Comment:
Those three are all functions in `HeuristicMapper::Node` and not `HeuristicMapper`. Since nodes do not have a back-reference to their mapper (if I'm not mistaken), they cannot access `results.config`.

Reply:
There actually is. Great idea, thank you! (There was even enough space for `admissibleHeuristic`)